<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="fr_Controls">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="cb_SampleRate">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>44.1 kHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>48 kHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>96 kHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>192 kHz</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lbl_SampleRate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Sample Rate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_BitDepth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Bit Depth:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_BitDepth">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>8-bit</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16-bit</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24-bit</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>32-bit</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btn_ExportFile">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Export File</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="fr_Macro2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbl_Volume">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_Volume">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QDial" name="dial_Volume">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>50</width>
       <height>64</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="fr_Buttons">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_Play">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Stop">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Pause">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="fr_Files">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_OpenFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Open File</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lbl_Title">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Titillium Web</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Synthesizer Testing Environment</string>
    </property>
   </widget>
   <widget class="QFrame" name="fr_Chart">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="le_MaxValue">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_DataPoint">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_MaxValue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max Value:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_DataPoint">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Datapoint #:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Chart">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Chart</string>
     </property>
    </widget>
   </widget>
   <zorder>fr_Macro2</zorder>
   <zorder>fr_Buttons</zorder>
   <zorder>fr_Controls</zorder>
   <zorder>fr_Files</zorder>
   <zorder>lbl_Title</zorder>
   <zorder>fr_Chart</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
